examples/src/main/java/kafka/examples/ExactlyOnceMessageProcessor.java

```diff
@@ -215,6 +238,39 @@ private long getRemainingRecords(KafkaConsumer<Integer, String> consumer) {
         }).sum();
     }
 
+    /**
+     * When we get a generic {@code KafkaException} while processing records, we retry up to {@code MAX_RETRIES} times.
+     * If we exceed this threshold, we log and error and move on to the next batch of records.
+     * In a real world application you may want to to send these records to a DLQ for further processing.
```

Comment:
1. we log [an] error
2. Should we spell the DLQ out in case users doesn't know it?